concepticondata/conceptlists/Morse-1996-375.tsv

```diff
@@ -0,0 +1,376 @@
+Morse-1996-375-233	233	corona	crown of head		1943	CROWN (OF HEAD)
+Morse-1996-375-234	234	dientes delanteros incisivos	front teeth		442	FRONT TOOTH (INCISOR)
+Morse-1996-375-235	235	punta de la lengua	tip tongue		443	TIP (OF TONGUE)
+Morse-1996-375-236	236	pelo largo	long hair		444	LONG HAIR
+Morse-1996-375-237	237	toda la garganta y cuello	around the neck			
+Morse-1996-375-238	238	nuez de la garganta	Adam’s apple		802	ADAM'S APPLE
+Morse-1996-375-239	239	espalda, parte superior	upper back		445	UPPER BACK
+Morse-1996-375-240	240	antebrazo	lower arm		446	LOWER ARM
+Morse-1996-375-241	241	muñeca	wrist		799	WRIST
+Morse-1996-375-242	242	la pierna inferior	lower leg		447	LOWER LEG
+Morse-1996-375-243	243	el pelo del cuerpo	body hair		189	HAIR (BODY)
+Morse-1996-375-244	244	estómago	stomach		819	STOMACH
+Morse-1996-375-245	245	intestinos	intestines	*	1991	INTESTINES
+Morse-1996-375-246	246	viejita	old woman		907	OLD WOMAN
+Morse-1996-375-247	247	nubes quietas	clouds at rest	*	1489	CLOUD
+Morse-1996-375-248	248	nubes de tormenta	storm clouds		448	STORM CLOUDS
+Morse-1996-375-249	249	chagra	corn field		449	CORN FIELD
+Morse-1996-375-250	250	quebrada	stream		947	STREAM
+Morse-1996-375-251	251	piedrecillas	pebbles		961	PEBBLES
+Morse-1996-375-252	252	rocas	huge rocks			
+Morse-1996-375-253	253	trocha	path		2252	PATH
+Morse-1996-375-254	254	persianas	venetian blinds			
+Morse-1996-375-255	255	tambo	shelter			
+Morse-1996-375-256	256	este	this		1214	THIS
+Morse-1996-375-257	257	aquel	that		78	THAT
+Morse-1996-375-258	258	quien	who		1235	WHO
+Morse-1996-375-259	259	que	what		1236	WHAT
+Morse-1996-375-260	260	no	not		1240	NOT
+Morse-1996-375-261	261	todos	all		98	ALL
+Morse-1996-375-262	262	muchos	many		1198	MANY
+Morse-1996-375-263	263	largo	long		1203	LONG
+Morse-1996-375-264	264	corteza	bark		1204	BARK
+Morse-1996-375-265	265	carne	flesh		2615	FLESH OR MEAT
+Morse-1996-375-266	266	sangre	blood		946	BLOOD
+Morse-1996-375-267	267	grase	grease		323	FAT (ORGANIC SUBSTANCE)
+Morse-1996-375-268	268	huevo	egg		744	EGG
+Morse-1996-375-269	269	cuerno	horn		1393	HORN (ANATOMY)
+Morse-1996-375-270	270	pluma	feather		1201	FEATHER
+Morse-1996-375-271	271	uña	claw		72	CLAW
+Morse-1996-375-272	272	barriga	belly		1251	BELLY
+Morse-1996-375-273	273	hígado	liver		1224	LIVER
+Morse-1996-375-274	274	beber	drink		1401	DRINK
+Morse-1996-375-275	275	comer	eat		1336	EAT
+Morse-1996-375-276	276	morder	bite		1403	BITE
+Morse-1996-375-277	277	ver	see		1409	SEE
+Morse-1996-375-278	278	oir	hear		1408	HEAR
+Morse-1996-375-279	279	saber	know		1410	KNOW (SOMETHING)
+Morse-1996-375-280	280	dormir	sleep		1585	SLEEP
+Morse-1996-375-281	281	morir	die		1494	DIE
+Morse-1996-375-282	282	matar	kill		1417	KILL
+Morse-1996-375-283	283	nadar	swim		1439	SWIM
+Morse-1996-375-284	284	volar	fly		1441	FLY (MOVE THROUGH AIR)
+Morse-1996-375-285	285	andar	walk		1443	WALK
+Morse-1996-375-286	286	venir	come		1446	COME
+Morse-1996-375-287	287	acostado	lie		601	LIE (MISLEAD)
```

Comment:
This one is more likely to mean 'to lie (in bed)' -> LIE (REST)

Reply:
Thank you, I totally missed that.

Reply:
yes, definitely.